### DISCUSSION TITLE
Cloning recipe

### DISCUSSION THREAD

**Discussion by rtalon** [nextcloud/cookbook#1651] [FAQ and user questions] [answered]
Hi !

Is it possible to clone / duplicate a recipe ?

Thank you !

**Comment by christianlupus** [comment]
Hello.

No, it is currently not possible to clone a recipe.

May I ask for your use-case?

**Comment by loadconfigs** [comment]
In my case it would usefull for optimizing a recipe, like trying to create the best WesternSauce-Clone. So you can easy create new recipes based of one existing.

**Reply by christianlupus** [reply]
Ok, I get it. This is currently not possible. I put a new feature request on the list of issues #1867

**Reply by loadconfigs** [reply]
Thank you very much!

**Reply by christianlupus** [reply, marked as answer]
With the newest releases of the app, this issue should be solved. There is the option to clone a recipe now.
